WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -348,7 +354,27 @@ character_event = {
 		completely_controls = c_innse_gall
 		
 		NOT = { religion_group = pagan_group }
-		culture_group = celtic
+		OR = {
+			culture_group = celtic
+			culture = romano-british
+			culture = saxon
+			culture = jute
+			culture = angle
+			culture = anglo-saxxon
+			culture = english
+			culture = norman
+			culture = old_saxon
+			culture = briton
+			culture = breton
+			has_landed_title = c_innse_gall
+		}
+
+		NOT = {
+			has_game_rule = {
+				name = supernatural_events
+				value = off
+			}
+		}
```

Comment:
Not for that reason, it is because it is for characters to convert to the religion which isn't necessary in our start-dates as they religion is alive.